docs/root/operations/cli.rst

```diff
@@ -8,8 +8,9 @@ following are the command line options that Envoy supports.
 
 .. option:: -c <path string>, --config-path <path string>
 
-  *(required)* The path to the v1 or v2 :ref:`JSON/YAML/proto3 configuration
-  file <config>`. This will be parsed as a :ref:`v2 bootstrap configuration file
+  *(optional)* The path to the v1 or v2 :ref:`JSON/YAML/proto3 configuration
+  file <config>`. If this flag is missing, --config-yaml is required.
```

Comment:
Can you `:option:` the --config-yaml on this line?

Reply:
Done.